assets/src/js/Components/archivePicker/ArchivePickerToolbar.js

```diff
@@ -0,0 +1,52 @@
+import {useMemo} from '@wordpress/element';
+import MultiSearchOption from './MultiSearchOption';
+import {useArchivePickerContext} from '../ArchivePicker';
+
+const {__} = wp.i18n;
+
+export default function ArchivePickerToolbar() {
+  const {
+    bulkSelect,
+    selectedImages,
+    selectedImagesAmount,
+    dispatch,
+    includeInWp,
+  } = useArchivePickerContext();
+
+  return useMemo(() => (
+    <div className="archive-picker-toolbar">
+      {!bulkSelect && <MultiSearchOption />}
+      <nav className="nav-bulk-select">
+        {bulkSelect && (
+          <button
+            disabled={false}
+            onClick={() => {
+              dispatch({type: 'DISABLE_BULK_SELECT'});
+            }}
+            type="button"
+            className="button"
+          >{__('Cancel', 'planet4-master-theme-backend')}</button>
+        )}
+
+        {bulkSelect ?
+          <button
+            onClick={async () => {
+              if (window.confirm(`You are about to import [${Object.keys(selectedImages).length}] photos to the media library. 'Cancel' to stop, 'OK' to import.`)) { // eslint-disable-line no-alert
```

Comment:
Why do we add `[]` around the amount of selected images here? I think it looks weird 🤔 
Also maybe we should add a verb to the last line, like `Press 'Cancel' to stop, 'OK' to import.`

<img width="464" alt="Screenshot 2023-08-01 at 11 13 41" src="https://github.com/greenpeace/planet4-master-theme/assets/6949075/cac4f1e3-ecf6-4e5e-a1f5-11cc6e4a4ccf">

Reply:
These 2 different questions are [answered here](https://www.figma.com/file/cnd1hDu5MnXD2kmGxfwH4N/Media-Archive-Wireframes-(Final-Beta)?type=design&node-id=220-7573&mode=dev).

About the `[]`, I copied the same text from the design. Do you think we should change it?

And regarding the modal, if you look at the bottom of design "Notes / Bahaviour", it says 
> "This is the native popup (in the bulk select/delete option in the media library), so I left that as is for now. I think we can revisit this at a later stage."   

What do you think?

Reply:
Yeah I'm fine with leaving is as it, it's not a big deal 😄